lib/modals/action_button.dart

```diff
@@ -16,27 +15,22 @@ class ActionButton extends StatelessWidget {
   final IconData iconData;
   final String title;
   final Function onTap;
-  Function? additionalCallback;
-
-  void setAdditionalCallback(Function callback) {
```

Comment:
Jeśli ten widget ma być Stateless to nie powinnien mieć setterów bo nie powinien sie zmieniać. Zamiast tego dodałem copyWith() żeby można był zrobić nowy ze zmienionym callbackiem